dashboard/test/mailers/previews/pd_teachercon1819_registration_mailer_preview.rb

```diff
@@ -0,0 +1,29 @@
+# This can be viewed on non-production environments at /rails/mailers/pd/teacher_application_mailer
```

Comment:
This is the wrong path. I think it'll be `/rails/mailers/pd/teachercon1819_registration_mailer`

Reply:
You can find it on localhost from `/rails/mailers`